src/content/reference/react/Fragment.md

```diff
@@ -19,29 +19,29 @@ title: <Fragment> (<>...</>)
 
 ---
 
-## Reference {/*reference*/}
+## Tham khảo {/*tham-khảo*/}
```

Comment:
Please refer to [style guide](https://github.com/reactjs/translations.react.dev/blob/main/style-guide.md#heading-ids), don't translate the heading ID since it can potentially break the link that used in different pages.

Reply:
This is auto-generated, I didn't change it

Reply:
But I will change it accordingly, thank you

Reply:
I mean, you should only translate the text, not the ID. Let's keep it like this: ## Tham khảo {/*reference*/}

Reply:
I mean I deleted the whole line and the heading Id got auto-generated, thank you on this one